.github/workflows/create-docker-images.yml

```diff
@@ -0,0 +1,29 @@
+# Creates a new docker image and pushes it to DockerHub when a version tag is created.
+name: create-docker-images
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  # Based on instructions from https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/
+  build-frontend-and-backend:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to docker hub
+        run: echo "${{ secrets.DOCKER_PASSWORD }}" | docker login -u "${{ secrets.DOCKER_USERNAME }}" --password-stdin
+      - name: 'Create env file'
+        run: |
+          touch .docker.env
+          echo DATABASE_URL=postgresql://${{secrets.POSTGRES_USER}}:${{secrets.POSTGRES_PASSWORD}}@db:5432/${{secrets.POSTGRES_DB}} >> .docker.env
```

Comment:
This shouldn't be necessary and it's also not secure. The environment variables can be set using the `--env` or `--env_file` options by the person running the container or via referencing variables in the docker compose file. See https://www.baeldung.com/ops/docker-container-environment-variables or https://www.baeldung.com/ops/docker-compose#managing-environment-variables for more details.